src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -113,6 +126,14 @@ public override bool Execute()
         GatherAotSourcesObjects(aotSources, aotObjects, extraSources, linkerArgs);
         GatherLinkerArgs(linkerArgs);
 
+        if (!UsesCustomRuntimeInitCallback)
+        {
+            WriteAutoInitializationFromTemplate();
+            extraSources.AppendLine("    autoinit.c");
+        }
+
+        GenerateAssembliesLoader();
```

Comment:
if exportedAssemblies == 0, we should error out the task.  Building a library without exports wouldn't be usable.

Reply:
good catch, added a throw inside `GatherAotSourcesObjects` right after looping over the assemblies to throw if no exported symbols were found.